homeassistant/helpers/llm.py

```diff
@@ -687,6 +687,14 @@ def _get_exposed_entities(
         if include_state:
             info["state"] = state.state
 
+            # Convert timestamp device_class states from UTC to local time
+            device_class = state.attributes.get("device_class")
+            if device_class == "timestamp" and state.state:
+                parsed_utc = dt_util.parse_datetime(state.state)
+                if parsed_utc is not None:
```

Comment:
Let's go ahead and inline if this can fit on one line

```suggestion
                if (parsed_utc := dt_util.parse_datetime(state.state)) is not None:
```